mdata/notifierKafka/notifierKafka.go

```diff
@@ -123,7 +123,16 @@ func (c *NotifierKafka) consumePartition(topic string, partition int32, currentO
 
 	messages := pc.Messages()
 	ticker := time.NewTicker(5 * time.Second)
-	startingUp := true
+
+	var startingUp bool
+	if bootTimeOffsets[partition] == 0 {
```

Comment:
What if there are 100k messages in the partition, but the first one we need to fetch is at offset 0?

Reply:
nevermind, that doesnt make sense.